tests/unittest/_torch/thop/parallel/test_indexer_topk_hist.py

```
@@ -0,0 +1,77 @@
+    """Run the ported-kernel correctness suite in a subprocess with the gate on;
+    require both a clean pass and evidence that the kernel armed."""
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    worker = os.path.join(current_dir, "indexer_topk_hist_tests.py")
+
+    process_env = os.environ.copy()
+    process_env["TRTLLM_DSA_TOPK_HIST"] = "1"
+    process_env["TLLM_LOG_LEVEL"] = "INFO"  # ensure the arming INFO log is emitted
+
+    # -s (no capture) lets the C++ stderr arming log flow through to our captured
+    # subprocess stderr; pytest would otherwise swallow it on success.
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)